python/dgl/nn/pytorch/network_emb.py

```diff
@@ -0,0 +1,220 @@
+"""Network Embedding NN modules"""
+# pylint: disable= invalid-name
+import torch
+from torch import nn
+from torch.nn import init
+from torch.utils.data import DataLoader
+import torch.nn.functional as F
+import numpy as np
+import tqdm
+from ...sampling import random_walk
+
+
+
+class Metapath2vec(nn.Module):
```

Comment:
```suggestion
class MetaPath2Vec(nn.Module):
```

We use the camel naming convention.